tests/Unit/Elliptic/DiscontinuousGalerkin/Test_DgOperator.cpp

```diff
@@ -0,0 +1,668 @@
+        DataVector{0.6964691855978616, 0.28613933495037946, 0.2268514535642031,
+                   0.5513147690828912};
+    Vars vars_rnd_midleft{4};
+    get(get<Var<Poisson::Tags::Field>>(vars_rnd_midleft)) =
+        DataVector{0.7194689697855631, 0.42310646012446096, 0.9807641983846155,
+                   0.6848297385848633};
+    Vars vars_rnd_midright{4};
+    get(get<Var<Poisson::Tags::Field>>(vars_rnd_midright)) =
+        DataVector{0.48093190148436094, 0.3921175181941505, 0.3431780161508694,
+                   0.7290497073840416};
+    Vars vars_rnd_right{4};
+    get(get<Var<Poisson::Tags::Field>>(vars_rnd_right)) =
+        DataVector{0.4385722446796244, 0.05967789660956835, 0.3980442553304314,
+                   0.7379954057320357};
+    PrimalFluxes expected_primal_fluxes_rnd_left{4};
+    get<0>(get<Var<::Tags::Flux<Poisson::Tags::Field, tmpl::size_t<1>,
+                                Frame::Inertial>>>(
+        expected_primal_fluxes_rnd_left)) =
+        DataVector{-4.027188081328469, -1.9207736184862605, 1.3653213194134493,
+                   3.3207435803332324};
+    PrimalFluxes expected_primal_fluxes_rnd_right{4};
+    get<0>(get<Var<::Tags::Flux<Poisson::Tags::Field, tmpl::size_t<1>,
+                                Frame::Inertial>>>(
+        expected_primal_fluxes_rnd_right)) =
+        DataVector{-5.281316261986065, -0.5513540594510129, 2.6634207403536934,
+                   1.9071387227305365};
+    OperatorVars expected_operator_vars_rnd_left{4};
+    get(get<DgOperatorAppliedTo<Var<Poisson::Tags::Field>>>(
+        expected_operator_vars_rnd_left)) =
+        DataVector{1785.7616039198579, 41.55242721423977, -41.54933376905333,
+                   -80.83628748342855};
+    OperatorVars expected_operator_vars_rnd_right{4};
+    get(get<DgOperatorAppliedTo<Var<Poisson::Tags::Field>>>(
+        expected_operator_vars_rnd_right)) =
+        DataVector{-299.00214422311404, -37.924582769790135, 2.1993037260176997,
+                   51.85418137198471};
+    test_dg_operator<system, true>(
+        domain_creator, penalty_parameter, analytic_solution,
+        analytic_solution_approx,
+        {{{{left_id, std::move(vars_rnd_left)},
+           {midleft_id, std::move(vars_rnd_midleft)},
+           {midright_id, std::move(vars_rnd_midright)},
+           {right_id, std::move(vars_rnd_right)}},
+          {{left_id, std::move(expected_primal_fluxes_rnd_left)},
+           {right_id, std::move(expected_primal_fluxes_rnd_right)}},
+          {{left_id, std::move(expected_operator_vars_rnd_left)},
+           {right_id, std::move(expected_operator_vars_rnd_right)}}}});
+  }
+  {
+    INFO("2D rectilinear");
+    using system =
+        Poisson::FirstOrderSystem<2, Poisson::Geometry::FlatCartesian>;
+    Poisson::Solutions::ProductOfSinusoids<2> analytic_solution{{{M_PI, M_PI}}};
+    Approx analytic_solution_approx =
+        Approx::custom().epsilon(1.e-2).margin(12.);
```

Comment:
The allowed errors for these tests are huge.  This margin is an order of magnitude larger than many of the values to be checked.

Reply:
I'll do a few more tests to look at the margins here.